ambry-api/src/main/java/com/github/ambry/named/StaleNamedResult.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2023 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.named;
+
+import java.sql.Timestamp;
+import java.util.Objects;
+
+
+/**
+ * Class to convey information about a successful stale named blob pulling {@link NamedBlobDb}.
+ */
+public class StaleNamedResult {
```

Comment:
nit: can you rename this to StaleNamedBlob

Reply:
Done